### PR TITLE
fix #78447 himovies.to

### LINKED ISSUE #78447: www5.himovies.to
### Issue URL (Ads)
 [https://www5.himovies.to/home](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww5.himovies.to%2Fhome)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Edge 89.0.774.57 (Chromium-based)
AdGuard driver: | WFP
Stealth mode: | disabled
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances
Userscripts: | https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js,<br>https://userscripts.adtidy.org/release/assistant/4.3/assistant.user.js,<br>https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js

### PR DESCRIPTION
**Pull request by wout-codes**
#78447

### CODE REVIEW

File: SpywareFilter/sections/tracking_servers.txt [hunk 1]
```diff
@@ -78,6 +78,7 @@
 ||tracdelight.io^$third-party
 ||goatcounter.com^$third-party
 ||gc.zgo.at^$third-party
+||tracker.idocdn.com:*/$third-party
```
Comment (Alex-302) [inline SpywareFilter/sections/tracking_servers.txt:81]: is `:*/` the necessary?

Comment (wout-codes) [reply SpywareFilter/sections/tracking_servers.txt:81]: Yes. AdGuard Apps.

Comment (wout-codes) [reply SpywareFilter/sections/tracking_servers.txt:81]: I changed it a little, but this is required `*`.

Comment (wout-codes) [reply SpywareFilter/sections/tracking_servers.txt:81]: Nope. I changed it.

Comment (wout-codes) [reply SpywareFilter/sections/tracking_servers.txt]: @Alex-302

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**